src/service/base/NativeMetaDb.cpp

```diff
@@ -40,23 +43,19 @@ void NativeMetaDb::open() {
 void NativeMetaDb::restore() noexcept {
   auto localFs = nebula::storage::makeFS("local", "");
   auto uriInfo = nebula::storage::parse(remote_);
-  if (uriInfo.schema == "s3") {
-    // sync all remote data into a local folder
-    auto fs = nebula::storage::makeFS("s3", uriInfo.host);
-
-    // find latest version indicated by the version file
-    const auto versionKey = fmt::format("{0}/version", uriInfo.path);
-    char version[32];
-    auto size = fs->read(versionKey, version, sizeof(version));
-    if (size > 0) {
-      // if existing the version, we download it to local to start with
-      auto snapshot = fmt::format("{0}/{1}", uriInfo.path, std::string_view(version, size));
-      if (fs->sync(snapshot, local_)) {
-        LOG(INFO) << "Reuse metadb saved at: " << snapshot;
-      }
+  // sync all remote data into a local folder
+  auto fs = nebula::storage::makeFS(uriInfo.schema, uriInfo.host);
```

Comment:
I feel like that this file system object can be a class member and don't need to be created every time?